mtp_api/apps/performance/view_dashboard.py

```diff
@@ -15,11 +15,17 @@
 
 
 def get_user_satisfaction():
-    yearly_data = requests.get(
-        'https://www.performance.service.gov.uk/data/send-prisoner-money/customer-satisfaction?'
-        'flatten=true&duration=1&period=year&collect=rating_1%3Asum&collect=rating_2%3Asum&collect=rating_3%3Asum&'
-        'collect=rating_4%3Asum&collect=rating_5%3Asum&collect=total%3Asum&format=json'
-    ).json()
+    try:
+        response = requests.get(
+            'https://www.performance.service.gov.uk/data/send-prisoner-money/customer-satisfaction?'
+            'flatten=true&duration=1&period=year&collect=rating_1%3Asum&collect=rating_2%3Asum&collect=rating_3%3Asum&'
+            'collect=rating_4%3Asum&collect=rating_5%3Asum&collect=total%3Asum&format=json'
+        )
+        response.raise_for_status()
+    except requests.RequestException:
+        return 'Not available'
```

Comment:
Does the client code use this returned value or it's just ignored/displayed directly?

If the client code doesn't do anything "special" with this (e.g. use as a key for a dictionary or what not) this should be fine.

Reply:
AFAICT this is set in a view context and displayed directly in the template so it should be fine 👌

Reply:
i did check, it just gets displayed in the template as-is. i used this pattern because, a few lines down, this function can return a string as an error message if no data was available